tests/dummy/app/components/subscription-feed.ts

```diff
@@ -47,6 +47,6 @@ export default class SubscriptionFeed extends Component<SubscriptionFeedArgs> {
       console.log('package received: ', JSON.stringify(event));
```

Comment:
No opinion on this, @upfluence/frontend ?

Reply:
So that we don't display this info if an end-user opens up the console ?
I don't think it's necessary ; we're not displaying any sensible data.

This specific log is in the dummy app which is just for test purposes.

Reply:
Could be a good cooldown subject though :)